portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,17 +27,23 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  List<String> data = new ArrayList<>(); 
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List<String> data = new ArrayList<>(); 
-    data.add("peanut");
-    data.add("cashew");
-    data.add("pistachio"); 
-
     Gson gson = new Gson();
     String json = gson.toJson(data); 
 
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String text = request.getParameter("form-comment"); 
+    if (text.length() != 0) {
```

Comment:
Ideally in the client, you can disable the submit button when the inputbox is empty :)

Otherwise there might be tons of post requests without any comment.

Reply:
OH that's so smart actually I didn't think about disabling the button! Thank you!